src/Microsoft.AspNet.Antiforgery/IAntiforgeryTokenValidator.cs

```diff
@@ -1,23 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System.Security.Claims;
-using Microsoft.AspNet.Http;
-
-namespace Microsoft.AspNet.Antiforgery
-{
-    // Provides an abstraction around something that can validate anti-XSRF tokens
-    public interface IAntiforgeryTokenValidator
```

Comment:
Combined with `IAntiforgeryTokenGenerator`. If you can generate a token, you probably can validate it too, and the implementation code already reflected that.